src/main/java/com/dev/cinema/Main.java

```diff
@@ -52,11 +54,10 @@ public static void main(String[] args) throws AuthenticationException {
 
         shoppingCartService.registerNewShoppingCart(user);
```

Comment:
let's call `register()` method instead

and then `login()`
wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block

Reply:
done